requirements.txt

```diff
@@ -4,8 +4,10 @@
 #
 #    pip-compile pyproject.toml
 #
-ansible==2.9.27
+ansible==2.10.7
```

Comment:
2.9.27 -> 2.10.x should be an upgrade that doesn't break compatibility (aside from the obvious package split issue), but I haven't taken the time yet to audit the subsequent changelogs to see what we'd need to update in e.g. ceph-cm-ansible. But 2.10 does fix 4/5 of the the security issues at https://github.com/ceph/teuthology/security/dependabot

I do think it's important to work to validate 5.x sooner rather than later.